vdr/vdr.go

```diff
@@ -243,3 +247,25 @@ func (r VDR) resolveControllerWithKey(doc did.Document) (did.Document, crypto.Ke
 
 	return did.Document{}, nil, fmt.Errorf("could not find capabilityInvocation key for updating the DID document: %w", err)
 }
+
+func patchContext(next did.Document) did.Document {
```

Comment:
will this be here forever? Or is temporary? When/how will it be removed?

Reply:
A `s/patchContext/withLDContext/` makes it explicit that you return a copy with changes, as opposed to a (pointer) setter.

Reply:
> will this be here forever? Or is temporary? When/how will it be removed?

Unknown, if in the future users are going to be responsible then the CRUD API should also do checks to prevent non-context members (like in VCs).